packages/pagebuilder/package.json

```diff
@@ -60,7 +60,7 @@
   },
   "pwa-studio": {
     "targets": {
-      "intercept": "./intercept"
+      "intercept": "./lib/intercept"
```

Comment:
Rearranged code to make sure that everything is in `./lib`, so that we don't accidentally fail to publish (hi @tjwiebell )